.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dcodeIO/setup-node-nvm@v5
+        with:
+          node-version: iojs
+      - run: npm install
+      - run: npm test
+        env:
```

Comment:
i ended up wrapping dcodeIO's action in `ljharb/actions/node/install`, which I use on 400 projects for node versions ranging from 0.4 to latest. I also have `ljharb/actions/node/matrix` to easily and dynamically create a matrix. You can also use `ljharb/actions/.github/workflows/node.yml` to encapsulate all of this in a single swoop.